src/posts/flexible-indexing/index.md

```diff
@@ -0,0 +1,218 @@
+---
+title: 'Xarray Indexes: Exciting new ways to slice and dice your data!'
+date: '2025-08-11'
+authors:
+  - name: Benoît Bovy
+    github: benbovy
+  - name: Scott Henderson
+    github: scottyhq
+  - name: Deepak Cherian
+    github: dcherian
+  - name: Justus Magin
+    github: keewis
+summary: 'An introduction to customizable coordinate-based data selection and alignment for more efficient handling of both traditional and more exotic data structures'
+---
+
+\_TLDR: Xarray>2025.6 has been through a major refactoring of its internals that makes coordinate-based data selection and alignment customizable, enabling more efficient handling of both traditional and more exotic data structures. In this post we highlight a few examples that take advantage of this new superpower!
```

Comment:
The TLDR looks a bit weird with the leading underscore after rendering, and I'd probably write the version comparison as `xarray>2025.06.0`. Is it truly `>` or rather `>=`, though?

Reply:
the version itself is fine, I just found the `>` a bit unusual, I would have used `>=` instead (and adjusted the version as necessary)